packages/babel-parser/test/fixtures/es2018/object-rest-spread/expression-rest-not-last-invalid/output.json

```diff
@@ -23,7 +22,7 @@
             "start":1,"end":13,"loc":{"start":{"line":1,"column":1,"index":1},"end":{"line":1,"column":13,"index":13}},
             "properties": [
               {
-                "type": "SpreadElement",
+                "type": "RestElement",
```

Comment:
This is a slight improvement, since it gives a better shape to the recovered AST (the input code is `({ ...rest, x } = {})`.